src/main/java/com/artipie/rpm/meta/XmlOthers.java

```diff
@@ -150,13 +150,27 @@ public Package version(final int eopch, final String ver, final String rel)
          *  be extracted from package headers. Check example RPM packages in test bin resources
          *  and `other.xml` file for this file in test resources.
          */
-        public Package changelog() throws XMLStreamException {
+        public Package changelog()
+            throws XMLStreamException {
             this.xml.writeStartElement("changelog");
             this.xml.writeCharacters("NOT_IMPLEMENTED");
             this.xml.writeEndElement();
             return this;
         }
 
+        /**
+         * Adds changelog tag to others.xml file.
+         * @param author Changelog author
+         * @param epoch Epoch seconds
+         * @param content Changelog content
+         * @return Self
+         * @throws XMLStreamException On error
+         */
+        public Package changelog(final String author, final int epoch, final String content)
+            throws XMLStreamException {
+            return this.changelog();
```

Comment:
@paulodamaso add a puzzle for this method, the changelog puzzle does not cover this.

Reply:
@Vatavuk please take a look at https://github.com/artipie/rpm-adapter/issues/82

Reply:
@paulodamaso but that puzzle is for method `changelog()` not `changelog(final String author, final int epoch, final String content)`

Reply:
@Vatavuk The idea is to implement changelog feature in `changelog(final String author, final int epoch, final String content)` and remove `changelog()` implementation, or make it call `changelog(final String author, final int epoch, final String content)`. The issue is already assigned to me, I just need this PR merged to avoid conflicts

Reply:
@paulodamaso ok